packages/components/src/card/card/README.md

```diff
@@ -1,54 +1,82 @@
 # Card
 
-Card provides a flexible and extensible content container.
+`Card` provides a flexible and extensible content container.
 
 ## Usage
 
-```jsx
-import { Card, CardBody } from '@wordpress/components';
+`Card` also provides a convenient set of [sub-components](#sub-components) such as `CardBody`, `CardHeader`, `CardFooter`, and more (see below).
 
-const Example = () => (
-	<Card>
-		<CardBody>...</CardBody>
-	</Card>
-);
+```jsx live
+import {
+	Card,
+	CardHeader,
+	CardBody,
+	CardFooter,
+	Text,
+	Heading,
+} from '@wordpress/components';
+
+function Example() {
+	return (
+		<Card>
+			<CardHeader>
+				<Heading size={ 4 }>Card Title</Heading>
+			</CardHeader>
+			<CardBody>
+				<Text>Card Content</Text>
+			</CardBody>
+			<CardFooter>
+				<Text>Card Footer</Text>
+			</CardFooter>
+		</Card>
+	);
+}
 ```
 
 ## Props
 
-### isBorderless
+### `elevation`: `number`
+
+Size of the elevation shadow, based on the Style system's elevation system. This may be helpful in highlighting certain content. For more information, check out [`Elevation`](/packages/componnents/src/elevation/README.md).
```

Comment:
```suggestion
Size of the elevation shadow, based on the Style system's elevation system. This may be helpful in highlighting certain content. For more information, check out [`Elevation`](/packages/components/src/elevation/README.md).
```